File: dsaRecent/LinearSearch/MinimumNumber.py
def minimumNumber(arr):
    min = arr[0]
    for i in range(0,len(arr)):
        if arr[i]<min:
            min = arr[i]
    return min

li = [19,124,9,23,5,3]
print(f"minimum number in list is {minimumNumber(li)}")

File: dsa/3LinearSearch/MinimumNumber.java
public class MinimumNumber {
    public static void main(String[] args) {
        int[] nums = { 9, 20, 35, 73, 46, 2, 99 };
        System.out.println("minimum number is "+ minimum(nums));
    }

    public static int minimum(int[] arr) {
        int min = arr[0];
        for (int i = 1; i < arr.length; i++) {
            if (arr[i] < min) {
                min = arr[i];
            }

        }
        return min;
    }
}
